rasa/core/brokers/kafka.py

```diff
@@ -20,6 +20,7 @@ def __init__(
         partition_by_sender: bool = False,
         sasl_username: Optional[Text] = None,
         sasl_password: Optional[Text] = None,
+        sasl_mechanism: Optional[Text] = "PLAIN",
```

Comment:
do we have a test which asserts this is correctly set on the class?

Reply:
Ah, not yet. Just added a test that checks it's been correctly set.